### PR TITLE
Improve squash quick start demo at install guide & Source map setting.

### PR DESCRIPTION
**Pull request by govargo**
**What this PR does / why we need it:**

This PR improve the squash quick guide for demo.
Fix typo and wrong installation way. 
And I added an explanation to the part that is hard to understand.

**Which issue this PR fixes**

Relation issue is here: https://github.com/solo-io/squash/issues/213.

**Special notes for your reviewer:** 
None.

### CODE REVIEW

File: squash/docs/overview/_index.md [hunk 1]
```diff
@@ -212,11 +212,19 @@ You should see a screen similar to the one below.
...
 The Next step is to tell the Squash extension the location of the `squashctl` CLI.
 
-Open **Settings** (File > Preferences > Settings) on Visual Studio Code, and then search for Squash. Once you find Squash, change the value of Path to point to the location of the CLI.
+Open **Settings** (Code > Preferences > Settings) on Visual Studio Code, and then search for Squash. Once you find Squash, change the value of Path to point to the location of the CLI.
 
 ![config](../images/vs-code-config-squash.png)
 
 If the target process was compiled from a different sourcepath, you should also [specify the remote path](/configuration/#source-code-mapping).
+In this quick start, if you choose go example2 app, you have to set “Squash: Remote Path” in VSCode User Settings.
+Please set "/home/yuval/go/src/github.com/solo-io/squash/contrib/example/" to “Squash: Remote Path”.
```
Comment (scranton) [inline squash/docs/overview/_index.md]: I'd flip the wording of this sentence around and use markdown code block, i.e.,

Please set `Squash: Remote Path` to `/home/yuval/go/src/github.com/solo-io/squash/contrib/example/`.

File: squash/docs/overview/_index.md [hunk 2]
```diff
@@ -212,11 +212,19 @@ You should see a screen similar to the one below.
...
+
+![vscode-remotepath](../images/vs-code-config-remote-path.png)
+
+If there is an error such as "Error on CreateBreakpoint: could not find /Users/<YourName>/squash/contrib/example/service1/main.go", 
+please correct its remote path in order to match "/home/yuval/go/src/github.com/solo-io/squash/contrib/example/service1/main.go".
```
Comment (scranton) [inline squash/docs/overview/_index.md]: Minor nit. We tend to use markdown backtick \` for most literals in the documentation, right or wrong, so for consistency helpful if you'd follow the same convention.

File: squash/docs/overview/_index.md (hunk 1 above)
Comment (govargo) [reply squash/docs/overview/_index.md]: I understand. Thank for your review!

File: squash/docs/overview/_index.md (hunk 2 above)
Comment (govargo) [reply squash/docs/overview/_index.md]: I understood too. 
I will use it from now on.

**Pull request merged**